lookout/style/typos/analyzer.py

```diff
@@ -107,6 +108,47 @@ def analyze(self, ptr_from: ReferencePointer, ptr_to: ReferencePointer,
                         comments.append(comment)
         return comments
 
+    @staticmethod
+    def reconstruct_identifier(tokenizer: TokenParser, pred_tokens: List[str], identifier: str) \
+            -> str:
+        """
+        Reconstruct identifier given predicted tokens and initial identifier.
+
+        :param tokenizer: tokenizer - instance of TokenParser.
+        :param pred_tokens: list of predicted tokens.
+        :param identifier: identifier.
+        :return: reconstructed identifier based on predicted tokens.
+        """
+        # setup required parameters
+        meta = tokenizer._metadata
```

Comment:
This is fine for now, but may I ask you to add these properties in sourced-ml. Thanks!